### PR TITLE
New version: Amazon.WorkspacesClient version 5.9.0

### PR DESCRIPTION
**Pull request by JamieSabbatella**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106520)

### CODE REVIEW

File: manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml
Comment (mdanish-kh) [inline manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml:8]:
```suggestion
InstallerType: wix
Scope: machine
```
Package installs machine-wide (for all users)

File: manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml
Comment (mdanish-kh) [inline manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml:22]:
```suggestion
  ProductCode: '{34A7E999-C65F-4297-A523-3B08C6C37352}'
  AppsAndFeaturesEntries:
    - DisplayVersion: 5.9.0.4129
```
Marketing version is `5.9.0`, but DisplayVersion written to registry (visible in Apps&Features) that WinGet uses to match upgrades is `5.9.0.4129` and should be mentioned in the manifest to avoid upgrade issues

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
Please try to retain as much metadata from the previous manifest as possible to avoid issues in upgrade scenario for users.

File: manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml
Comment (JamieSabbatella) [reply manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml:8]: Apologies these was just done with quick update with YamlCreate. I assumed it would keep original info

Comment (mdanish-kh) [reply manifests/a/Amazon/WorkspacesClient/5.9.0/Amazon.WorkspacesClient.installer.yaml:8]: Well it should've done so if you used its `Quick Update` functionality. Maybe you didn't update your fork and `git pull` into your local repo as YamlCreate uses the latest 'local' manifest as the base manifest for update. winget-create on the other hand, always fetches the latest manifest from the upstream repo so you won't likely experience that problem there

**Comment by mdanish-kh** [comment]
@JamieSabbatella Just an fyi: The PR has been manually validated and is waiting on you to resolve the comments before merge.

**Comment by stephengillie** [comment]
> @JamieSabbatella Just an fyi: The PR has been manually validated and is waiting on you to resolve the comments before merge.

Yes - sorry about these `Internal-Error-Dynamic-Scan` errors. They're caused by Automated Validation pipeline interference, and the package did pass our alternate Manual Validation pipeline.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**
